pcweb/pages/demos_on_landing_page/auth/auth.py

```diff
@@ -0,0 +1,117 @@
+import reflex as rx
+
+def auth_form():
+    return rx.flex(
+        rx.flex(
+            rx.text(
+                "Create an account",
+                font_size="24px",
+                line_height="2em",
+                weight="bold",
+                color="#FFFFFF",
+                align="center",
+                height="32px",
+                width="350px",
+            ),
+            rx.box(
+                height="8px",
+                wdith="350px",
+            ),
```

Comment:
`wdith`

but also, why use a box spacer here instead of something like `margin_bottom="8px"` on the element above?